api/v1alpha1/httproute_types.go

```diff
@@ -140,7 +140,6 @@ type HTTPRouteMatch struct {
 // within a known namespace.
 //
 // +k8s:deepcopy-gen=false
-// +protobuf=false
```

Comment:
@howardjohn, I had to delete this specific marker or else `go-to-protobuf` failed with the following output:
```
topological order io
topological order bytes
topological order go/ast
topological order encoding/base64
topological order strings
topological order k8s.io/apimachinery/pkg/fields
topological order sync
topological order go/token
topological order k8s.io/apimachinery/pkg/watch
topological order k8s.io/apimachinery/pkg/selection
topological order k8s.io/apimachinery/pkg/conversion/queryparams
topological order strconv
topological order github.com/google/gofuzz
topological order go/parser
topological order reflect
topological order sync/atomic
topological order encoding/json
topological order github.com/gogo/protobuf/sortkeys
topological order fmt
topological order k8s.io/apimachinery/pkg/util/runtime
topological order k8s.io/apimachinery/pkg/util/naming
topological order time
topological order k8s.io/apimachinery/pkg/util/json
topological order github.com/gogo/protobuf/proto
topological order k8s.io/apimachinery/pkg/api/resource
topological order net/url
topological order k8s.io/apimachinery/pkg/types
topological order k8s.io/klog
topological order k8s.io/apimachinery/pkg/util/errors
topological order k8s.io/apimachinery/pkg/conversion
topological order math/bits
topological order errors
topological order go/doc
topological order math
topological order k8s.io/apimachinery/pkg/runtime/schema
topological order k8s.io/apimachinery/pkg/util/intstr
topological order unsafe
topological order k8s.io/apimachinery/pkg/labels
topological order os
topological order k8s.io/apimachinery/pkg/util/sets
topological order k8s.io/apimachinery/pkg/runtime
topological order k8s.io/apimachinery/pkg/apis/meta/v1
topological order sigs.k8s.io/controller-runtime/pkg/scheme
topological order k8s.io/api/core/v1
topological order sigs.k8s.io/service-apis/api/v1alpha1
2020/02/27 23:33:32 sigs.k8s.io/service-apis/api/v1alpha1/generated.proto:142:12: "LocalObjectReference" is not defined.
sigs.k8s.io/service-apis/api/v1alpha1/generated.proto:300:12: "LocalObjectReference" is not defined.
sigs.k8s.io/service-apis/api/v1alpha1/generated.proto:312:12: "LocalObjectReference" is not defined.
sigs.k8s.io/service-apis/api/v1alpha1/generated.proto:334:12: "LocalObjectReference" is not defined.
sigs.k8s.io/service-apis/api/v1alpha1/generated.proto:371:12: "LocalObjectReference" is not defined.
sigs.k8s.io/service-apis/api/v1alpha1/generated.proto:424:12: "LocalObjectReference" is not defined.
sigs.k8s.io/service-apis/api/v1alpha1/generated.proto:513:12: "LocalObjectReference" is not defined.
sigs.k8s.io/service-apis/api/v1alpha1/generated.proto:586:12: "LocalObjectReference" is not defined.
sigs.k8s.io/service-apis/api/v1alpha1/generated.proto: warning: Import sigs.k8s.io/controller-runtime/pkg/scheme/generated.proto but not used.
sigs.k8s.io/service-apis/api/v1alpha1/generated.proto: warning: Import k8s.io/apimachinery/pkg/runtime/schema/generated.proto but not used.
sigs.k8s.io/service-apis/api/v1alpha1/generated.proto: warning: Import k8s.io/api/core/v1/generated.proto but not used.
2020/02/27 23:33:32 protoc -I . -I /home/mmasters/src -I /home/mmasters/src/sigs.k8s.io/service-apis/vendor -I /home/mmasters/src/sigs.k8s.io/service-apis/third_party/protobuf --gogo_out=/home/mmasters/src /home/mmasters/src/sigs.k8s.io/service-apis/api/v1alpha1/generated.proto
2020/02/27 23:33:32 Unable to generate protoc on k8s.io.service_apis.api.v1alpha1: exit status 1
exit status 1
```
I have not tried using the container image, but I cannot see how that would make a difference.  Anyway, `go-to-protobuf` was happy whether I deleted this marker or all of them, so I deleted all of them.

Reply:
Ok as long as it builds it should be good, maybe your change fixed the issue we were working around. I am not sure the CI would fail if it doesn't build though, maybe be worth running a quick `go build ./... `?

Reply:
`go build ./...` succeeded (no output; exit status 0).